### PR TITLE
Update ping health and add code examples

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: v4/source/system.yaml [hunk 1]
```diff
@@ -2,18 +2,31 @@
     get:
       tags:
         - system
-      summary: Ping the Server
+      summary: Check system health
       description: |
-        Receive a simple ping response.
+        Check if the system is health or unhealthy based on the configuration setting `GoRoutineHealthThreshold`.
```
Comment (jwilander) [inline v4/source/system.yaml]: Can we update this to "Check if the server is up and healthy based on the configuration setting \`GoRoutineHealthThreshold\`. If \`GoRoutineHealthThreshold\` and the number of goroutines on the server exceeds that threshold the server is considered unhealthy. If \`GoRoutineHealthThreshold\` is not set or the number of goroutines is below the threshold the server is considered healthy."

File: v4/source/system.yaml [hunk 2]
```diff
@@ -2,18 +2,31 @@
...
+        __Minimum server version__: 3.10
         ##### Permissions
-        No permissions required.
+        Must have `manage_system` permission
```
Comment (jwilander) [inline v4/source/system.yaml]: There is no permission required

File: v4/source/system.yaml [hunk 3]
```diff
@@ -2,18 +2,31 @@
...
       responses:
         '200':
-          description: Ping successful
+          description: Status of the system
           schema:
```
Comment (jwilander) [inline v4/source/system.yaml]: The schema for 200 didn't change, I think this needs to be under '500'

**Comment by jwilander** [comment]
Can you also submit the code samples as a separate PR?

File: v4/source/system.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

File: v4/source/system.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

File: v4/source/system.yaml (hunk 3 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

**Pull request merged**